main.tf

```diff
@@ -101,6 +105,20 @@ resource "aws_iam_role_policy" "datafy" {
           }
         }
       },
+      {
+        "Effect" : var.permissions_level == "Sensor" ? "Deny" : "Allow",
+        "Action" : [
+          "ec2:CreateTags"
```

Comment:
cool.. i guess its the only way..
can you please combine this with the permission bellow of the `DeleteTags` as its exactly the same